nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -163,9 +163,9 @@ def transform_model(
                 converted_zero_point = opset.convert(zero_point_const, const_dtype)
                 converted_const = opset.subtract(converted_const, converted_zero_point)
 
-            scale_const = opset.constant(
-                compressed_weight.scale.data, dtype=const_dtype, name=f"{const_node_name}/scale"
-            )
+            scale_const = opset.constant(compressed_weight.scale.data, dtype="float16", name=f"{const_node_name}/scale")
+            if const_dtype != "float16":
```

Comment:
The test is necessary for these changes.

Reply:
Added